### DISCUSSION TITLE
Legend

### DISCUSSION THREAD

**Discussion by mblau54937** [MMRIZE/MMM-CalendarExt3#39] [Ideas]
Is there a way to display a color matching legend for this calendar?

**Comment by eouia** [comment]
Can you show/describe detail what you want? example or description of the case,...

**Comment by mblau54937** [comment]
I was hoping to have some time of legend that shows the labels for the calendars along with the associated color. Id really like it see the Calendar name, then a box next to it with the color or a color surrounding the calendar title.

example:
![image](https://user-images.githubusercontent.com/83167352/198858659-9a38a175-7159-4b18-8441-6f09444ab5b8.png)

**Comment by eouia** [comment]
Ah, good idea. I'll consider it. But there is one issue. Like your example, the legend would usually be located in another region area to the main CX3 module. It means, it is better to be a different module to respect MM's behaviours. Anyway, when I have a time I'll provide.

**Comment by mblau54937** [comment]
Thank you. I would prefer it on the bottoms, that was just an example of someone did a custom jpg and used a different module, but I’m personally not a fan of that look.

**Comment by eouia** [comment]
I've updated to show legend. Check it.
